tvtk/pyface/ui/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -271,50 +177,17 @@ def _get_win_id(self):
     def Finalize(self):
         '''
         Call internal cleanup method on VTK objects
+
+        New in VTK 6.0.0
         '''
         self._RenderWindow.Finalize()
 
-    def CreateTimer(self, obj, evt):
-        self._Timer.start(10)
-
-    def DestroyTimer(self, obj, evt):
-        self._Timer.stop()
-        return 1
-
-    def TimerEvent(self):
-        self._Iren.TimerEvent()
-
-    def CursorChangedEvent(self, obj, evt):
-        """Called when the CursorChangedEvent fires on the render window."""
-        # This indirection is needed since when the event fires, the current
-        # cursor is not yet set so we defer this by which time the current
-        # cursor should have been set.
-        QTimer.singleShot(0, self.ShowCursor)
-
-    def HideCursor(self):
-        """Hides the cursor."""
-        self.setCursor(Qt.BlankCursor)
-
-    def ShowCursor(self):
-        """Shows the cursor."""
-        vtk_cursor = self._Iren.GetRenderWindow().GetCurrentCursor()
-        qt_cursor = self._CURSOR_MAP.get(vtk_cursor, Qt.ArrowCursor)
-        self.setCursor(qt_cursor)
-
-    def closeEvent(self, evt):
-        self.Finalize()
-
-    def sizeHint(self):
-        return QSize(400, 400)
-
-    def paintEngine(self):
-        return None
-
     def paintEvent(self, ev):
+        super(QVTKRenderWindowInteractor, self).paintEvent(ev)
         self._RenderWindow.Render()
```

Comment:
I am not sure if this `self._RenderWindow.Render()` is only required by tvtk Scene.  If so, then it should be moved to tvtk/pyface/ui/qt4/scene.py